pages/Documents/ConversationalAI/ConversationBuilder/Bots/bot-groups.md

```diff
@@ -25,13 +25,13 @@ It's common to organize bots into groups based on business function. You could t
 
 Note the following about the bots dashboard when bot groups exist:
 
-<img class="fancyimage" style="width:1000px" src="img/ConvoBuilder/botGroups_dashboard.png">
+<img class="fancyimage" style="width:1000px" src="img/ConvoBuilder/botGroups_dashboard.png" alt="The Bots dashboard with several key features highlighted">
 
 1. The bot group name. Click this bar to show/hide the bots in the group.
 2. The list of bots in the group. Click any bot name to open the bot in the Dialog Editor.
 3. The **Collaboration** icon lets you know that collaboration ([automatic transfers](conversation-builder-bots-bot-to-bot-transfers.html#automatic-transfers-via-bot-group)) is enabled for the group.
 4. No icon is displayed when collaboration is disabled for the group.
-5. Click the <img style="width:25px" src="img/ConvoBuilder/icon_ellipsis_vertical_light.png"> icon to access the group menu, which provides options for editing and deleting the group.
+5. Click the <img style="width:25px" src="img/ConvoBuilder/icon_ellipsis_vertical_light.png" alt="Three-dot icon"> icon to access the group menu, which provides options for editing and deleting the group.
```

Comment:
Unless we want to go for consistency, here we could switch to “three dots”. For a sighted user this would not change anything and be fine, but for a screen reader, “three dots icon” would be better than “three-dot icon icon”. (Suggesting “Three dots” because standalone, this in turn is better than “Three-dot”.)